hazelcast/src/main/java/com/hazelcast/transaction/impl/xa/XATransactionImpl.java

```diff
@@ -78,8 +78,8 @@
     private final SerializableXID xid;
     private final String txOwnerUuid;
 
-    private final List<TransactionRecord> txLogs = new LinkedList<TransactionRecord>();
-    private final Map<Object, TransactionRecord> txLogMap = new HashMap<Object, TransactionRecord>();
+    private final List<TransactionRecord> records = new LinkedList<TransactionRecord>();
```

Comment:
Isn't this a common term? They also use the term "log record" here: https://en.wikipedia.org/wiki/Transaction_log